include/sys/fm/fs/zfs.h

```diff
@@ -78,6 +78,8 @@ extern "C" {
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_READ_ERRORS	"vdev_read_errors"
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_WRITE_ERRORS "vdev_write_errors"
 #define	FM_EREPORT_PAYLOAD_ZFS_VDEV_CKSUM_ERRORS "vdev_cksum_errors"
+#define	FM_EREPORT_PAYLOAD_ZFS_VDEV_CKSUM_N	"vdev_cksum_n"
+#define	FM_EREPORT_PAYLOAD_ZFS_VDEV_CKSUM_T	"vdev_cksum_t"
```

Comment:
Yes, that was the plan, we just wanted to get some feedback before we did them all.

While we are here, is there any reason not to promote "space_on_remove" to vdev properties too?

Reply:
Initially "spare_on_remove" struck me as more of a pool-wide property, but upon further reflection it would be nice to have the per-vdev granularity.  That would let you nicely control which removed vdevs the spare would kick-in for.  The only leaves "remove_timeout" which I think would make sense as a vdev property to.

Reply:
I've updated this pull request to include support for vdev_io_n and vdev_io_t.